analytics_test.go

```diff
@@ -30,14 +30,17 @@ func TestAnalytics_GetAggregatedIncidentData(t *testing.T) {
 	}
 
 	bytesAnalyticsResponse, err := json.Marshal(analyticsResponse)
+	testErrCheck(t, "json.Marshal()", "", err)
```

Comment:
The ineffective assignment linter caught that we weren't handling the `err` value above; this is absolutely a bug.